dnplab/hydrationGUI.py

```diff
@@ -167,19 +167,30 @@ def __init__(self):
         self.optwidthCheckbox.resize(100, 20)
```

Comment:
I didn't review changes to this script. Please let me know if you have specific things that I can help to check.

Reply:
Ok, I think everything is in order. All tests pass, the example runs, and the GUI works fine